Integrate-Kobiton.md

```diff
@@ -0,0 +1,273 @@
+
+    Replace `docker` section with the selected image
+    
+    ```yamls
+    docker:
+      - image: circleci/node:latest
+    ```
+
+2. Modify CircleCI configuration file
+    
+    To update `npm` to the latest version, in `steps` section, add the following lines under `checkout`
+
+    ```yaml
+    - run:
+        name: update-npm
+        command: 'sudo npm install -g npm@latest'
+    ```
+
+    Because the provided scripts use 3rd party libraries, thus we will have to install them using npm.
+
+    In `steps` section, add the following lines under `update-npm` section
+
+    ```yaml
+    - run:
+        name: npm-restore-deps
+        command: npm install
+    ```
+
+    To execute the chosen script, in `steps` section, add the following lines under `npm-restore-deps` section
+
+    ```yaml
+    - run:
+        name: android-app-test
+        command: npm run <YOUR_SCRIPT_FILENAME>
+    ```
+
+    The contents of `config.yml` should look like this
+    
+    ```yaml
+    version: 2
+    jobs:
+    build:
+        working_directory: ~/sample
+        docker:
+            - image: circleci/node:latest
+        steps:
+            - checkout
+            - run:
+                name: update-npm
+                command: 'sudo npm install -g npm@latest'
+            - run:
+                name: npm-restore-deps
+                command: npm install
+            - run:
+                name: android-app-test
+                command: npm run <YOUR_SCRIPT_FILENAME>
+    ```
+
+### 5. Executing the automated test script
+
+Simply push all changes to your repository and CircleCI will execute the script. To view the execution progress in CircleCI, follow these steps below
+
+1. Go to [CircleCI dashboard](https://circleci.com/dashboard). You will see status of the test
+
+- If the scripts are being executed
+
+![alt text](./assets/test-executing.png)
+
+- If your script(s) have been successfully executed without any error
+
+![alt text](./assets/test-success.png)
+
+- Otherwise
+
+![alt text](./assets/test-failed.png)
+
+2. Click on the title of the test to see it's details
+
+![alt text](./assets/test-title.png)
+
+3. Test details should look like this
+
+![alt text](./assets/test-details.png)
+
+### 6. Viewing Kobiton test session information
+
+**On Kobiton website**
+
+1. Go to [https://portal.kobiton.com/sessions](https://portal.kobiton.com/sessions), login with your Kobiton account.
+2. You will see your executed sessions. Click on any session you want to view its details.
+
+![alt text](./assets/session.png)
+
+**Using REST API**
+
+Besides viewing on website, your session details can also be retreived using Kobiton's REST API. This 
+
+1. Get session ID :
+
+    Go to your test result on CircleCI, expand the `android-app-test` section, look for `RESPONSE` section. You will see the session ID (marked **light blue**)
```

Comment:
I think this step is too technical since users have to look for the console log. It'd be better to tell them to go to Kobiton website to get the session ID.